### PR TITLE
Timber1900.WebDL version v10.2

### PR DESCRIPTION
**Pull request by Timber1900**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20911)

### CODE REVIEW

**Comment by palenshus** [comment]
Looks like this is actually an x64 app, not x86

File: manifests/t/Timber1900/WebDL/v10.2/Timber1900.WebDL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Timber1900.WebDL
+PackageVersion: v10.2
+Installers:
+- Architecture: x86
```
Comment (palenshus) [inline manifests/t/Timber1900/WebDL/v10.2/Timber1900.WebDL.installer.yaml:7]: This appears to actually be an x64 app

**Review by palenshus — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]
@palenshus when you request a change, only you are allowed to approve it. Next time sugges the change so the author can do commit changes.

**Review by palenshus — APPROVED** [review]

**Comment by palenshus** [comment]
> @palenshus Union Palenshus FTE when you request a change, only you are allowed to approve it. Next time sugges the change so the author can do commit changes.

Ah, I didn't realize that, thanks for the tip!

**Pull request merged**